### ISSUE TITLE
407 though PyCharm Proxy settings are correct

### ISSUE THREAD

**Issue by johannes-mathes** [aws/aws-toolkit-jetbrains#2999]
Hi Team,

I got repeatedly 407 Errors while using the AWS Toolkit in PyCharm. Sometimes, the 407 is gone, but suddenly it comes back.
I am behind a corporate proxy which uses authentication. My proxy settings are set, including Auth in PyCharm.
---
Toolkit: AWS Toolkit For JetBrains 1.36-212
OS: Mac OS X 11.4
IDE: PyCharm Community Edition 212

**Comment by johannes-mathes** [comment]
If I go in proxy preferences and do check connection, the 407 is gone for a while.

**Comment by johannes-mathes** [comment]
Any news on this issue?

**Comment by breedloj** [comment]
There appears to have been an IntelliJ IDE bug working with auth enabled proxies: https://youtrack.jetbrains.com/issue/IDEA-276949

This has been addressed on 2021.3.1. Prior to the update I was able to reproduce this issue. After updating I am now able to successfully connect through a proxy that requires a username and password. The AWS Toolkit is inheriting the proxy settings configured in IntellIJ and properly connecting. I am also able to successfully connect using the latest version of PyCharm.
